src/solvers/dg/dg.jl

```diff
@@ -53,6 +53,18 @@ end
   return nothing
 end
 
+
+"""
+    calc_error_norms([func=(u,equation)->u,] solver, t)
+
+Calculate the discrete L2 and L∞ errors of `func` applied to the conservative variables of
+the problem encapsulated by `solver` at time `t`, where `func` is called as `func(u, equation)`.
+"""
+function calc_error_norms end
```

Comment:
If this is not DG specific, wouldn't it be better placed in `solvers/solvers.jl`?

Reply:
Done.